source/screens/caseScreens/CaseOverview.tsx

```diff
@@ -40,6 +40,7 @@ import { Form } from "../../types/FormTypes";
 const {
   ACTIVE_RANDOM_CHECK_REQUIRED_VIVA,
   ACTIVE_COMPLETION_REQUIRED_VIVA,
+  ACTIVE_COMPLETION_SUBMITTED,
```

Comment:
We need to add `ACTIVE_RANDOM_CHECK_SUBMITTED` as well. This is the good old 'Stickprovskontroll'

Reply:
Created a separate task for that one 
https://app.clickup.com/t/1vzckax